CMake/kwiver-depends-Qt.cmake

```diff
@@ -12,5 +12,16 @@ option( KWIVER_ENABLE_QT
   )
 
 if( KWIVER_ENABLE_QT )
-  find_package( Qt5 5.10 REQUIRED COMPONENTS Core Gui )
+  find_package( Qt5 5.10 REQUIRED COMPONENTS Core Gui Widgets Xml)
```

Comment:
*Probably* not a big deal, but would it be worthwhile only requiring Widgets and Xml if `KWIVER_ENABLE_QT_EXT` is enabled?